src/components/list/list.js

```diff
@@ -327,7 +363,9 @@ function mdListItemDirective($mdAria, $mdConstant, $mdUtil, $timeout) {
       function copyAttributes(item, wrapper) {
         var copiedAttrs = $mdUtil.prefixer([
           'ng-if', 'ng-click', 'ng-dblclick', 'aria-label', 'ng-disabled', 'ui-sref',
-          'href', 'ng-href', 'target', 'ng-attr-ui-sref', 'ui-sref-opts'
+          'href', 'ng-href', 'target', 'ng-attr-ui-sref', 'ui-sref-opts',
+          // Custom attributes for the md-button component.
+          'md-no-focus-style'
```

Comment:
This seems REALLY weird; why not "class="md-no-focus" instead of `md-no-focus-style`?